modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPortalPart2.testcase

```diff
@@ -225,11 +225,73 @@ definition {
     }
 
     @description = "Automation ID: LPS-151804 | Test Summary: Verify if '- Remove Activation Key' is working if there are at least 2 sets of Server Details on the form"
+	@ignore = "true"
 	@priority = "4"
 	test IfRemoveActivationKeyIsWorking {
-        
-		// TODO LPS-151804 CP2NewKeyGenerationPortalPart2 archive pending implementation
 
+		// Needs refactor since it depends on example NASA ICE account
+
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "${testSiteName}");
+		}
+
+		task ("Given I fulfill entire flow up to the second phase for Generate New License") {
+			CP2ActivationKeys.generateKeyFlow();
+		}
+
+        task ("Given I fulfill the Environment field") {
+			CP2ActivationKeys.fulfillGenerateActivationKeyTextField(
+				content = "Environment Test Name",
+				field = "Environment Name");
+		}
+
+		task ("And the '- Remove Activation Key' button should not preset") {
```

Comment:
Hi @luanakarla, there is a typo, can you please rename this task to "And the '- Remove Activation Key' button should be not present"?

Reply:
Yes, I hadn't seen it. Maybe the more correct way would be [ And the '- Remove Activation Key' button should not be present ] Because we are using the auxiliary should when this auxiliary is used in the negative it is shouldn't in the case of the sentence should not is not contracted, if it were contracted the verb would still come after the not: button shouldn't be present. So should not be present is correct. Okay?

Reply:
Yes, I hadn't seen it. Maybe the more correct way would be [ And the '- Remove Activation Key' button should not be present ] okay?

Reply:
Yes, you're right! 😅